test/integration/README.md

```diff
@@ -0,0 +1,59 @@
+# Tests for Mozilla's Lockbox WebExtension (discopane)
+
+Thank you for checking out Mozilla's Lockbox WebExtension!
+This repository contains Selenium tests used to test the discovery pane.
+
+Note: These tests are written in python and use pytest as the runner.
+
+## How to run the tests locally
+
+### Clone the repository
+If you have cloned this project already then you can skip this, otherwise you'll need to clone this repo using Git.
+If you do not know how to clone a GitHub repository, check out this
+[help page][git-clone] from GitHub.
+
+If you think you would like to contribute to the tests by writing or maintaining them in the future,
+it would be a good idea to create a fork of this repository first, and then clone that.
+GitHub also has great instructions for [forking a repository][git-fork].
+
+### Build the webextension
+
+The tests expect the webextension to be built. If not you will receive an error stating that the
+addon or webextension is not found. 
+
+```bash
+npm run prepackage
```

Comment:
the `prepackage` script should get run by npm automatically when running `npm run package` below.